include/openmc/zplane_partitioner.h

```diff
@@ -0,0 +1,37 @@
+#ifndef OPENMC_ZPLANE_PARTITIONER_H
+#define OPENMC_ZPLANE_PARTITIONER_H
+
+#include "universe.h"
+#include <set>
+#include <vector>
+
+namespace openmc {
+
+using namespace std;
+
+class ZPlanePartitioner : public UniversePartitioner {
```

Comment:
Does the ATR example have a good amount of x- and y-planes? If not, could you send an example input that does? I think it would be useful when testing out the efficiency of x- and y-plane partitioners.